src/main/java/com/rapid7/client/dcerpc/objects/RPCReferentConformantArray.java

```diff
@@ -33,6 +34,7 @@ public RPCReferentConformantArray(T[] array) {
 
     @Override
     public void unmarshalEntity(PacketInput in) throws IOException {
+        in.align(Alignment.FOUR);
```

Comment:
Technically this alignment is variable. The alignment of a conformant array is `Max[size_of(length), size_of(entity)]`

Reply:
For example this would not work for a `long[]`

Reply:
Unfortunately you may need another abstract method to report the `Alignment` of the data type the array contains.

Reply:
@tosmun-r7 
This is a referent conformant array, unless we are using NDR64, otherwise this should be 4 bytes, no?

Reply:
For long[] we need a separate class.

Reply:
Ah yes, for a reference to an array that's right.